exercises/querying-workflows/solution/signalclient.py

```diff
@@ -0,0 +1,14 @@
+import asyncio
+import time
+
+from temporalio.client import Client
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_workflow_handle("queries")
+    await handle.signal("submit_greeting", "User 1")
+    time.sleep(1)
+    await handle.signal("exit")
```

Comment:
Often these types of samples/exercises would wait for handle result here too and show workflow result